### ISSUE TITLE
output in JSON

### ISSUE THREAD

**Issue by FinlayDaG33k** [monitoring-plugins/monitoring-plugins#1416]
This is more an enhancement than an issue, but please add a function to give the output in JSON... this makes is alot easier to work with in for example PhP

**Comment by riskersen** [comment]
Hi, what output do you mean? Do you mean in general for all plugins?

**Comment by FinlayDaG33k** [comment]
Yes, as it is right now quite a pain on the brain to use the output of the plugins in for example PHP scripts... I know you can use something my friends call "tokenization" but having the ability to have the script output in JSON is way user-friendlier in my opinion.

**Comment by RincewindsHat** [comment]
this was implemented partially (for most of the `C`-based plugins) with version 3.0.0
The rest will likely follow.
